shacl12-core/index.html

```diff
@@ -1294,7 +1294,7 @@ <h3>Constraints, Parameters and Constraint Components</h3>
 }</pre></div></div>
 					</aside>
 					<p>
-						Constraint components are associated with <dfn data-lt="validator|validators">validators</dfn>, which provide instructions (for example expressed via SPARQL queries)
+						Constraint components are associated with <dfn>validators</dfn>, which provide instructions (for example expressed via SPARQL queries)
```

Comment:
The change here goes backwards. *Either* both singular and plural must be included in the `data-lt` attribute, or only the singular or plural which is not in the `<dfn>...</dfn>`. Some of the changes in this PR dropped one of these, and some of the changes added one. I can see no reason why inconsistency would deliver the desired result.
```suggestion
						Constraint components are associated with <dfn data-lt="validator|validators">validators</dfn>, which provide instructions (for example expressed via SPARQL queries)
```